toolkit-base/clearfix.scss

```diff
@@ -0,0 +1,27 @@
+/* ======== //
+// CLEARFIX //
+// ======== */
+
+
+/*
+// Behavior: Prevent a container’s height from collapsing when floating child elements.
+// Dependencies: None
+// Browser Support: http://caniuse.com/#feat=css-gencontent
+// Further Reading: http://nicolasgallagher.com/micro-clearfix-hack
+*/
+
+
+.clearfix {
+
+	&:before,
+	&:after {
+
+		content: " ";
+		display: table;
+	}
+
+	&:after {
+
+		clear: both;
```

Comment:
That’s just how the clearfix works.

```
/**
 * For modern browsers
 * 1. The space content is one way to avoid an Opera bug when the
 *    contenteditable attribute is included anywhere else in the document.
 *    Otherwise it causes space to appear at the top and bottom of elements
 *    that are clearfixed.
 * 2. The use of `table` rather than `block` is only necessary if using
 *    `:before` to contain the top-margins of child elements.
 */
.cf:before,
.cf:after {
    content: " "; /* 1 */
    display: table; /* 2 */
}

.cf:after {
    clear: both;
}

/**
 * For IE 6/7 only
 * Include this rule to trigger hasLayout and contain floats.
 */
.cf {
    *zoom: 1;
}
```